### PR TITLE
Implement blacklisting feature

### PR DESCRIPTION
**Pull request by puiterwijk**
Fixes: #374
Signed-off-by: Patrick Uiterwijk <puiterwijk@redhat.com>

### CODE REVIEW

File: pkgdb2/default_config.py [hunk 1]
```diff
@@ -52,6 +52,9 @@
 # List the ACL which are auto-approved (don't need reviewing)
 AUTO_APPROVE = ['watchcommits', 'watchbugzilla']
 
+### Blacklisted items
+REQUEST_BLACKLIST = ['fedora']
```
Comment (pypingou) [inline pkgdb2/default_config.py]: We probably want to make this a dict allowing to specify blacklisted package per namespace

**Comment by pypingou** [comment]
Thanks for working on this, I've had it on my list so that makes my list shorted :)

File: utility/pkgdb2.cfg.sample [hunk 2]
```diff
@@ -19,6 +19,9 @@ DB_URL='sqlite:////var/tmp/pkgdb2_dev.sqlite'
 ### pages
 ITEMS_PER_PAGE = 50
 
+### Blacklisted items
```
Comment (pypingou) [inline utility/pkgdb2.cfg.sample]: Could we provide one or more example of what the content should look like? :)

Comment (puiterwijk) [inline utility/pkgdb2.cfg.sample]: Done, one line below :smile:

Comment (pypingou) [inline utility/pkgdb2.cfg.sample]: But maybe we could include it in the doc, something like:
~~~~
### Blacklisted items (following the format: ``namespage/packagename``)
~~~~
Thoughts?

**Comment by pypingou** [comment]
Looks good to me, thanks!

**Pull request merged**